lib/harness/baker.js

```diff
@@ -140,9 +140,16 @@ class BakerConnector extends SshConnector {
         }
     }
 
-    async ready(){
-        // TODO: check if baker env is ready to be used (status == 'running')
-        return;
+    async ready() 
+    {
+        const env_address = process.cwd();
+        const context = { bakerPath: env_address };
```

Comment:
I will update `ready()` to get `context` as a parameter -- `ready(context)`. These two first lines are duplicate code and we already have it in `index.js`.